### ISSUE TITLE
NullPointerException in onSaveInstanceState

### ISSUE THREAD

**Issue by esh0** [sevar83/indeterminate-checkbox#1]
There is a case, when we get NPE in onSaveInstanceState. This is calling 
`ss.indeterminate = this.getState().booleanValue();`
Which for indeterminate state will try to evaluate booleanValue on null.

Stacktrace:

```
E/AndroidRuntime: FATAL EXCEPTION: main
                  Process: com.coresuite.coresuitemobile, PID: 4145
                  java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.Boolean.booleanValue()' on a null object reference
                      at com.buildware.widget.indeterm.IndeterminateCheckBox.onSaveInstanceState(IndeterminateCheckBox.java:218)
                      at android.view.View.dispatchSaveInstanceState(View.java:13550)
                      at android.view.ViewGroup.dispatchSaveInstanceState(ViewGroup.java:2865)
                      at android.view.ViewGroup.dispatchSaveInstanceState(ViewGroup.java:2865)
                      at android.view.ViewGroup.dispatchSaveInstanceState(ViewGroup.java:2865)
                      at android.view.ViewGroup.dispatchSaveInstanceState(ViewGroup.java:2865)
                      at android.view.View.saveHierarchyState(View.java:13533)
                      at android.support.v4.app.FragmentManagerImpl.saveFragmentViewState(FragmentManager.java:1850)
                      at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1180)
                      at android.support.v4.app.FragmentManagerImpl.removeFragment(FragmentManager.java:1387)
                      at android.support.v4.app.BackStackRecord.run(BackStackRecord.java:758)
                      at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:1677)
                      at android.support.v4.app.FragmentManagerImpl$1.run(FragmentManager.java:536)
                      at android.os.Handler.handleCallback(Handler.java:739)
                      at android.os.Handler.dispatchMessage(Handler.java:95)
                      at android.os.Looper.loop(Looper.java:135)
                      at android.app.ActivityThread.main(ActivityThread.java:5221)
                      at java.lang.reflect.Method.invoke(Native Method)
                      at java.lang.reflect.Method.invoke(Method.java:372)
                      at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:899)
                      at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:694)
```

**Comment by svkostadinoff** [comment]
Thanks for reporting. I'll see what's going on this evening.

**Comment by pecet66** [comment]
I have the same issue.

**Comment by jhandei** [comment]
I have the same issue.

It occurs when you minimize the app while fragment with checkbox in indeterminate state.

**Comment by svkostadinoff** [comment]
Fixed in 1.0.5. Feel free to report everything suspicious.
